lib/cocoapods/command/env.rb

```diff
@@ -0,0 +1,65 @@
+require 'cocoapods/user_interface/error_report'
+
+module Pod
+  class Command
+    class Env < Command
+      def run
+        puts "
+#{'――― MARKDOWN TEMPLATE ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――'.reversed}
```

Comment:
The template itself shouldn't be duplicated, and the header and footer shouldn't be printed by the command, only by the error report.